### PR TITLE
Align credentials stores part 12

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Relates:[ link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5274)

## Description
Change the type of credentials, from type 4 to type 9.

## Minimum version of Cortex XSOAR
- [x] 6.5.0

## Does it break backward compatibility?
  - [x] No

### CODE REVIEW

**Comment by maimorag** [comment]
Notes regarding ServiceDestPlus:
1. Test playbook fails, there is an open issue regarding it. Therefore the changes in the code are not causing it. [link to Jirra issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-1459)
2. The args Client ID and Client Secret type:4 are in the combined param credentials_client of type:9 -> Is it ok?

**Comment by Shellyber** [comment]
> Notes regarding ServiceDestPlus:
> 
> 1. Test playbook fails, there is an open issue regarding it. Therefore the changes in the code are not causing it. [link to Jirra issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-1459)
> 2. The args Client ID and Client Secret type:4 are in the combined param credentials_client of type:9 -> Is it ok?

1. Did you try to fix the test? understand why it fails? 
2. Yes, that's ok.

**Comment by maimorag** [comment]
@Shellyber @JudahSchwartz The test fails since the secret key is expired. I looking for a way to renew it

**Comment by maimorag** [comment]
a refresh_token had been generated in order to fix the playbook issue [link to Jirra issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-1459)

File: Packs/Looker/Integrations/Looker/Looker.py [hunk 1]
```diff
@@ -401,6 +402,8 @@ def create_look(query_id, space_id, look_title, look_description=""):
 
 def main():
     LOG('Command being called is %s' % (demisto.command()))
+    if not(CLIENT_ID and CLIENT_SECRET):
+        return_error('API3 Client ID and Secret must be provided.')
```
Comment (Shellyber) [inline Packs/Looker/Integrations/Looker/Looker.py:406]: Use raise Exception

**Review by Shellyber — APPROVED** [review]

**Pull request merged**